data/core.yaml

```
@@ -1894,6 +1894,11 @@ en:
       tip: "Find relations with missing or incorrect member roles"
       multipolygon:
         reference: "Multipolygon members must have an inner or outer role."
+    missing_start_date:
+      title: Empty start date
+      feature:
+        message: "Start date shouldn't be empty"
```

For consistency, the title should be in Title Case. Also, I accidentally resolved https://github.com/OpenHistoricalMap/iD/pull/193#discussion_r1528031938 but meant to point out that we should call it “missing” rather than “empty” in the UI too, not just internally.

```suggestion
      title: Missing Start Dates
      feature:
        message: "Find features that are missing start dates"
```

---

I mixed up the `tip` and `message`. Fixed in eeec11e52b002e770b7537b409f550c1d7c3432f.